debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/metrics/SqlServerPartitionMetricsMXBean.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.sqlserver.metrics;
+
+/**
+ * Metrics scoped to a source partition that are common for both snapshot and streaming change event sources
+ */
+public interface SqlServerPartitionMetricsMXBean {
+
+    String getLastEvent();
+
+    long getMilliSecondsSinceLastEvent();
+
+    long getTotalNumberOfEventsSeen();
+
+    long getNumberOfEventsFiltered();
+
+    long getNumberOfErroneousEvents();
+
+    /**
+     * @deprecated Superseded by the 'Captured Tables' metric. Use {@link #getCapturedTables()}.
+     * Scheduled for removal in a future release.
+     */
+    @Deprecated
+    String[] getMonitoredTables();
```

Comment:
Given this is a new MBean, I don't think we need this; let's just focus on the new one below.

On the interface in general though: I still think this could just be a generic `PartitionMetricsMXBean`, none of its metrics look specific to SQL Server. Something to discuss in the call tomorrow?

Reply:
> On the interface in general though: I still think this could just be a generic PartitionMetricsMXBean, none of its metrics look specific to SQL Server. Something to discuss in the call tomorrow?

Yeah, let's discuss it. I may have answered it in https://github.com/debezium/debezium/pull/2946#discussion_r799918088.